08-rails/reference/.rails-template.rb

```diff
@@ -10,64 +12,87 @@
 
 FIGLET
 
-# Make $(document).ready work as expected, despite turbolinks weirdness
-gem "jquery-turbolinks"
+unless API_MODE
+  # jQuery is cool
+  gem "jquery-rails"
 
-gem "jquery-rails"
+  # Make $(document).ready work as expected, despite turbolinks weirdness
+  gem "jquery-turbolinks"
+  # Bootstrap CSS Library
+  gem "bootstrap"
+  gem "sass-rails"
+end
 
 gem_group :development, :test do
   # Use pry for rails console, enable binding.pry
   gem "pry-rails"
 end
 
 gem_group :development do
-  # Improve the error message you get in the browser
-  gem "better_errors"
-
-  # Nice interactive terminal when an exception happens
-  gem "binding_of_caller"
-
-  # Automatically run our tests
-  gem "guard"
-  gem "guard-minitest"
   # Hot New Debugging Gems
   gem 'debase', '>= 0.2.4.1'
   gem 'ruby-debug-ide', '>= 0.7.0'
+end
 
+unless API_MODE
+  gem_group :development do
+    # Improve the error message you get in the browser
+    gem "better_errors"
+
+    # Nice interactive terminal when an exception happens
+    gem "binding_of_caller"
+
+    # Automatically run our tests
+    gem "guard"
+    gem "guard-minitest"
```

Comment:
Would guard be ok in an API?

Reply:
Oh, yeah.  Moved that by accident.  Good catch!